examples/3dtiles_google.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Itowns - 3d-tiles example</title>
+
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="css/example.css">
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <div id="description">
+            <p><b>Feature Information:</b></p>
+            <div id="featureInfo"></div>
+        </div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/GUI/LoadingScreen.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script src="js/3dTilesHelper.js"></script>
+        <script type="text/javascript">
+            const placement = {
+                coord: new itowns.Coordinates("EPSG:4326", 4.83116, 45.756026),
+                range: 25000,
+            };
+
+            // iTowns namespace defined here
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            var view = new itowns.GlobeView(viewerDiv, placement);
+            view.camera.camera3D.near = 5;
+            setupLoadingScreen(viewerDiv, view);
+
+            var menuGlobe = new GuiTools('menuDiv', view, 300);
+
+            itowns.Fetcher.json('./layers/JSONLayers/OPENSM.json').then(function _(config) {
+                config.source = new itowns.TMSSource(config.source);
+                var layer = new itowns.ColorLayer('Ortho', config);
+                view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
+            });
+
+            // Create a new Layer Google 3d-tiles
+            // -------------------------------------------
+
+            const extensions = new itowns.C3DTExtensions();
+
+            extensions.registerExtension("3DTILES_content_gltf", {
+                [itowns.C3DTilesTypes.batchtable]: itowns.C3DTGLTFExtension,
+            });
+
+            itowns.enableDracoLoader("./libs/draco/");
+
+            var $3dTilesLayerDiscreteLOD = new itowns.C3DTilesLayer(
+                "3d-google-tiles-discrete-lod-a",
+                {
+                    name: "DiscreteGoogleLOD",
+                    sseThreshold: 16, // default value
+                    source: new itowns.C3DTilesGoogleSource({
+                        url: "https://tile.googleapis.com/v1/3dtiles/root.json",
+                        urlParameters: {
+                            key: "YOUR_API_KEY",
```

Comment:
It would be nice to support a URL parameter `key` to set the google cloud API key ! And maybe an input which sets this parameter?